### ISSUE TITLE
headers shortcuts

### ISSUE THREAD

**Issue by DoctorSubtilis** [Zettlr/Zettlr#3455]
### Description

I wonder if is my fault, but I can't figure out how to set shortcuts for h1, h2, h3 (as ctrl+1, ctrl+2, ctrl+3 and so on).

### Reproducing

With ctrl+1 nothing happens....

### Zettlr Version

Stable (most recent version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

KDE Neon

**Comment by nathanlesage** [comment]
Are you looking for shortcuts to insert headings of this type?

In any case since you phrase this issue like a question, I'm going to remove the bug label here. Please choose the appropriate issue types, and ask questions in the discussions section.

**Comment by DoctorSubtilis** [comment]
> Are you looking for shortcuts to insert headings of this type?

Yes for the titles or headers (# h1, ## h2, ### h3 and so on)

> In any case since you phrase this issue like a question, I'm going to remove the bug label here. Please choose the appropriate issue types, and ask questions in the discussions section.

Sorry!  
What should I do now?

**Comment by nathanlesage** [comment]
I'll rephrase this as a feature request, don't worry!

But in general it makes sense to read the documentation. If something is not mentioned in there which you would like to have (such as Shortcuts to transform headers), then it is a feature request. If something is mentioned in there but does not work as advertised, it is a bug. But in the latter case, first asking around in the GitHub discussions section or on Discord helps, because sometimes "not working" just boils down to a simple misunderstanding how it is supposed to work.

In general, please try to open issues here only if you are absolutely certain that you would like to request a feature or a bug, since that keeps our work here easier to do! But for now, sorry for all the explanation, welcome to the community! :)

**Comment by DoctorSubtilis** [comment]
Thank you!

**Comment by do-not-perceive-me** [comment]
I made shortcuts for inserting the headers with Autocorrect in Preferences.  

When I type 'h5' it replaces it with '#####'

**Comment by agentlibre** [comment]
> I made shortcuts for inserting the headers with Autocorrect in Preferences.
> 
> When I type 'h5' it replaces it with '#####'.
> 
> I also added them into snippets and named it as 5, so when I type ':5', it shows up with extra line breaks around it.
> 
> ```
> 
> ##### ${1:heading}
> 
> $0
> ```

I fail to understand how this is a replacement. The idea is to be able to use a shortcut next to the word or sentence, even from the right side, then have it converted to a heading.
